images/win/Windows2022-Readme.md

```diff
@@ -18,24 +16,24 @@
 - Bash 4.4.23(1)-release
 - Go 1.16.10
 - Julia 1.6.4
-- Kotlin 1.6.0
+- Kotlin 1.5.32
```

Comment:
Version downgrade is related to an API response (https://api.github.com/repos/JetBrains/kotlin/releases/latest) which is used for Kotlin installation. It will be fixed for the next rollout.